views/layout.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Layout</title>
+    
+  </head>
+  <body>
+    <h1>This Land is MY LAND</h1>
+    <h2><a href="/apartments">All Apartments</a></h2>
+    <h3><a href="/tenants">All Tenants</a></h3>
+    <%= yield %>
```

Comment:
Sinatra places all of the other views inside of this `<%= yield %>`. This means that you no longer need to have the `<!DOCTYPE html> <html> yada yada` stuff in any of the other views